source/adapters/cuda/command_buffer.cpp

```diff
@@ -242,7 +242,7 @@ static ur_result_t enqueueCommandBufferFillHelper(
     if ((PatternSize == 1) || (PatternSize == 2) || (PatternSize == 4)) {
       CUDA_MEMSET_NODE_PARAMS NodeParams = {};
       NodeParams.dst = DstPtr;
-      NodeParams.elementSize = PatternSize;
+      NodeParams.elementSize = static_cast<unsigned int>(PatternSize);
```

Comment:
I'm guessing that the variables (`PatternSize`, `gridDims`, `blockDims` and friends) are casted from `size_t`, that seems more suitable type than `unsigned int`. Perhaps, outside of the scope of this PR, but it feels like adjusting the types in node params structs would be a more elegant solution.

Reply:
Actually, maybe not, native (cuda) calls seem to prefer [`unsigned int`](https://github.com/oneapi-src/unified-runtime/pull/2100/files#diff-9dcf8810f13f5d1a1049e13dc684ccc23bb4739a8c3efdb1ae46e64475f5cc18R484). It's probably not worth trying to align all the different apis, never mind.

Reply:
Yeah its probably an impossible job having all the types align.